### DISCUSSION TITLE
Change the title on the home page?

### DISCUSSION THREAD

**Discussion by MatthK** [searxng/searxng#2539] [Q&A] [answered]
I tried to change the "SearXNG" title on the home page to a different name.

My instance is in a docker container. I opened a shell of the container with `docker exec -it container sh`. I then did a `vi /usr/local/searxng/searx/templates/simple/index.html` and replaced the SearXNG between the <h1></h1> tags with the name I chose. I saved the changes, exited and restarted the container.

When I browse to http://localhost:8080/ it still shows the big and blue "SearXNG" above the search field. However, when I do an inspect of the page, it does show my name instead of the SearXNG. What am I missing? How can I change the name on the home page?

**Comment by return42** [comment, marked as answer]
You can set the [instance_name:  "SearXNG"](https://docs.searxng.org/admin/settings/settings_general.html) in your local settings.yml.

**Reply by MatthK** [reply]
Thanks for the prompt reply. I have set that already, but that didn't change anything. But then I found the solution. I had to create a PNG (and SVG) file and copy that into the container as follows:

```
docker exec -it container sh
cp /etc/searxng/searxng.png /usr/local/searxng/searx/static/themes/simple/img/
cp /etc/searxng/searxng.svg /usr/local/searxng/searx/static/themes/simple/img/
```

I now have "my name" at the home page.
